fuse/plugins/detector_physics/s1_photon_hits.py

```diff
@@ -160,3 +160,7 @@ def get_n_photons(self, n_photons, positions):
         n_photon_hits = self.rng.binomial(n=n_photons, p=ly)
 
         return n_photon_hits
+
+# For here:
+# 1. get_n_photons is run multiple times, so the np.squeeze should be done when initializing the correction maps.
+# 2. s1 pattern map?
```

Comment:
`np.squeeze` is quite fast or? At least in the tests  with 20k `ly` values (corresponding to a file of 2k Pb214 events) it took just O(500 ns). I think we can keep it that way for now.

Reply:
I think I have accidentally left these comments :( These can be removed.